helm/flowforge/README.md

```diff
@@ -215,3 +215,82 @@ readinessProbe:
   successThreshold: 1
   failureThreshold: 3
 ```
+
+## Upgrading Chart
+
+### Unreleased
+
+Following database-related values have been removed from the chart:
+
+```
+forgedbName
+forge.dbUsername
+forge.dbPassword
+forge.postgres.host
+forge.postgres.port
+```
+
+All database configuration is maintained in the `postgresql` configuration block.
+Before upgrading to this version, please ensure that the [`postgresql`](#postgresql) block is correctly configured.
```

Comment:
I think we need to explicitly say that the `dbUsername` and `dbPassword` become `postgesql.auth.username` and `postgrsql.auth.password`

Also the I think we need a small update to the Database section at the start of the README to point out that these values are used for both local postgresql db and if an external DB is used.

Reply:
Added,